os/board/rtl8730e/src/component/soc/amebad2/app/monitor/rom/Make.defs

```diff
@@ -49,7 +49,7 @@
 #
 ############################################################################
 
-CSRCS += shell_rom.c \
-       monitor_ap.c
+CSRCS += # shell_rom.c \
+       # monitor_ap.c
```

Comment:
Since it's not used, maybe we can remove this file?

Reply:
done, please check~